src/dsc/Microsoft.PowerToys.Configure/examples/allSettings.dsc.yaml

```diff
@@ -0,0 +1,270 @@
+properties:
+  resources:
+    - resource: PowerToysConfigure
+      directives:
+        description: Configure PowerToys
+      settings:
+        AlwaysOnTop:
+          Hotkey: "<string>"
```

Comment:
Lots of "<string>" in here. If we're adding samples, we should do manual samples if the automated samples contain mostly trash.